tests/conftest.py

```diff
@@ -577,36 +316,35 @@ def assert_side_effects_invoked(side_effects_contract, side_effects_trigger, n=1
     return assert_side_effects_invoked
 
 
-@pytest.fixture
-def get_logs(w3):
-    def get_logs(tx_hash, c, event_name):
-        tx_receipt = w3.eth.get_transaction_receipt(tx_hash)
-        return c._classic_contract.events[event_name]().process_receipt(tx_receipt)
-
-    return get_logs
-
-
 @pytest.fixture(scope="module")
-def tx_failed(tester):
+def tx_failed(env):
```

Comment:
should probably be renamed since there is no longer a transaction object, but we could just leave as a comment (since the scope of this PR is already huge!)